### PR TITLE
Improve test coverage for calendar.tsx

### PR DESCRIPTION
**Pull request by adi-uchiha**
**What kind of change does this PR introduce?**

**Issue Number:**
Closes: #550 
Fixes #

**Did you add tests for your changes?**
YESSSSSSSSSSSSSS

**Snapshots/Videos:**
chrome_I8HRVaw6Qm

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
   "scripts": {
     "serve": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --coverage",
```
Comment (Rumone) [inline package.json]: Why was the coverage flag removed here?

Comment (adi-uchiha) [reply package.json]: @Rumone because it was not there, I added it for my convenience on previous commit

Comment (Rumone) [reply package.json]: Oh okay

**Comment by Rumone** [comment]
Screenshot 2023-03-13 at 12 14 27 AM

Based on the [link](https://app.codecov.io/gh/PalisadoesFoundation/talawa-admin/blob/develop/src/components/EventCalendar/Calendar.tsx) the initial test coverage was **88.16%** and currently it is **88.15%**

**Comment by adi-uchiha** [comment]
@Rumone sometimes it shows coverage of frist commit that's why I added the screenshot.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**